README.adoc

```diff
@@ -118,6 +118,33 @@ assert(!"rubocop-minitest" == (actual))
 refute_equal("rubocop-minitest", actual)
 ----
 
+=== Assert Same [[assert-same]]
+
+Use `assert_same` instead of `assert` with `equal?`.
+
```

Comment:
It may be convenient for users to write a `NOTE` about the difference in comparison method with `assert_equal`.

Reply:
Suggestion:

> Use `assert_same` only when there is a need to compare by identity. Otherwise, use `assert_equal`.